### ISSUE TITLE
Add accessibility tags in TinyMCE

### ISSUE THREAD

**Issue by gioacchino54** [joomla/joomla-cms#46815]
### Expected result
The TinyMCE editor needs some buttons to add specific tags such as < abbr >.
This is because not everyone knows how to use the code-side editor, whereas with a button, every user could do so, which would also make the backend a little more accessible.
Thank you.

### Actual result
No one

### Device and system info

* What version of Joomla were you using?
4.x - 5.x - 6.x

* What device were you using? (eg, phone, laptop)
Desktop

* What web browser were you using? (eg, Chrome, Safari)
Any

* If you were using assistive technology, please let us know the device or software you were using (eg VoiceOver, JAWS, sip-and-puff device, etc)

### Additional comments

If you know which [WCAG](https://www.w3.org/TR/WCAG/) or [ATAG](https://www.w3.org/TR/ATAG20/) criteria this violates, please indicate it here:

**Comment by chmst** [comment]
https://github.com/joomla/joomla-cms/pull/46820

**Comment by brianteeman** [comment]
Pretty sure I already did that some time ago. Will check when I get back to my pc tomorrow

**Comment by brianteeman** [comment]
anything else needed
